### PR TITLE
[stdlib] Remove FileCheck in test_aarch64_target.mojo

### PR DESCRIPTION
**Pull request by ilhamfp**
https://github.com/modularml/mojo/issues/2024

### CODE REVIEW

File: stdlib/test/sys/test_aarch64_target.mojo
Comment (gabrieldemarmiesse) [inline stdlib/test/sys/test_aarch64_target.mojo:26]: You can remove the comments `# CHECK...`

Comment (ilhamfp) [reply stdlib/test/sys/test_aarch64_target.mojo:25]: Thanks! Was eyeing this ticket before realizing you have worked on most of it. I don't think it's fair for me to take this one :) I'm closing it

Comment (ilhamfp) [reply stdlib/test/sys/test_aarch64_target.mojo:25]: Thanks! I was eyeing this ticket before realizing you have worked on most of it. I don't think it's fair for me to take this one :) 

I think I should close it...

Comment (ilhamfp) [reply stdlib/test/sys/test_aarch64_target.mojo:25]: Thanks! I was eyeing this ticket before realizing you have worked on most of it. I don't think it's fair for me to take this one :) 

I think I should close it and let you take the full credit...

Comment (gabrieldemarmiesse) [reply stdlib/test/sys/test_aarch64_target.mojo:25]: There is no need for you to close this pull request. Credit isn't important. You can remove the few comments that are in the code and this pull request should be good to merge

Comment (ilhamfp) [reply stdlib/test/sys/test_aarch64_target.mojo:25]: Ah, that's so nice of you. Removed the comments and opened the pull request again.

**Review by JoeLoser — APPROVED** [review]

**Comment by JoeLoser** [comment]
Looks good, but you'll need to rebase with the latest `nightly` branch so the code on your PR picks up to use the latest compiler in CI checks.

**Comment by gabrieldemarmiesse** [comment]
@JoeLoser if I may suggest another useful github feature: 
In the settings of the repo there is this option:
image

If you enable it, then it will always display this button in PRs for both maintainers and contributors:
image

This simplifies greatly the contribution process for people who are less familiar with git, and for maintainers, it can allow a quick rebase to make sure the CI is ok with the latest changes in nightly before merging

**Comment by JoeLoser** [comment]
> @JoeLoser if I may suggest another useful github feature: In the settings of the repo there is this option: image
> 
> If you enable it, then it will always display this button in PRs for both maintainers and contributors: image
> 
> This simplifies greatly the contribution process for people who are less familiar with git (using git with forks is slightly more complicated than without any fork), and for maintainers, it can allow a quick rebase to make sure the CI is ok with the latest changes in nightly before merging

Nice.  Just turned that option on in the settings - thanks!

**Review by JoeLoser — APPROVED** [review]
Thanks for the contribution — look good to me!

**Pull request merged**
